how-to/synonym-expansion.md

```diff
@@ -0,0 +1,150 @@
+---
+title: How to use synonym expansion
+---
+
+## How to use synonym expansion
+
+We need to make a new table for registering synonyms to search synonym.
+
+We explain how to register synonyms, delete synonyms, update synonyms, search synonyms, and make a table for register synonyms.
+
+### How to make a new table to register synonyms
+
+We make a table to register synonyms as below.
+
+```sql
+CREATE TABLE synonyms (
+  term text PRIMARY KEY,
+  synonyms text[]
+);
+
+CREATE INDEX synonyms_search ON synonyms USING pgroonga (term pgroonga.text_term_search_ops_v2);
+```
+
+We can get a term that is registered in `synonyms` with a key that is registered in `term`. 
+
+For example, if we want to also match "display" when specifying "window" as the search keyword, register "window" to `term` and register "window" and "display" to `synonyms`.
+(We attention to need also register target synonym term to `synonyms`.)
```

Comment:
`Note that we also need register...`
Is better to notice paying attention to? WDYT?

Reply:
I agree with you.
I'll fix to use Note.